read_roi/_read_roi.py

```diff
@@ -316,9 +316,13 @@ def extract_basic_roi_data(data):
         overlayFontSize = get_uint16(data, hdr2Offset + HEADER_OFFSET['OVERLAY_FONT_SIZE'])
         imageOpacity = get_byte(data, hdr2Offset + HEADER_OFFSET['IMAGE_OPACITY'])
         imageSize = get_uint32(data, hdr2Offset + HEADER_OFFSET['IMAGE_SIZE'])
+        # abb, read group from file
+        group = get_byte(data, hdr2Offset + HEADER_OFFSET['AVAILABLE_BYTE1'])
         logging.debug("Entering in hdr2Offset")
 
-    roi_props = (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size)
+    # abb, append group to tuple
+    #roi_props = (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size)
+    roi_props = (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size, group)
```

Comment:
No need for the comment, or to include the deleted line as a comment.
```suggestion
    roi_props = (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size, group)
```